RFS/src/main/java/com/rfs/cms/OpenSearchWorkCoordinator.java

```diff
@@ -297,7 +297,21 @@ DocumentModificationResult getResult(AbstractedHttpClient.AbstractHttpResponse r
         }
         final var resultDoc = objectMapper.readTree(response.getPayloadStream());
         var resultStr = resultDoc.path(RESULT_OPENSSEARCH_FIELD_NAME).textValue();
-        return DocumentModificationResult.parse(resultStr);
+        try {
+            return DocumentModificationResult.parse(resultStr);
+        } catch (Exception e) {
+            log.atWarn().setCause(e).setMessage(() -> "Caught exception while parsing the result").log();
+            log.atWarn().setMessage(() -> {
```

Comment:
Technically, the applications _had_ these logs before because wire logging was enabled for all work coordination requests.  Since this is such a critical need, having a JUnit test that confirms that the server output is in at log4j output at least when WARN is enabled would be great.  See ResultsToLogsConsumerTest in the replayer for an example of how to rig a test with JUnit.

Reply:
Done!  Thanks for the reminder.